hangups/auth.py

```diff
@@ -20,7 +20,7 @@
 import mechanicalsoup
 import requests
 
-from hangups import version
+from . import version
```

Comment:
I'd prefer to continue using absolute imports. PEP8 recommends them for most cases:

> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path)

> However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose